config/database.yml

```diff
@@ -21,6 +21,8 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password: io
```

Comment:
oh weird, ok. I'll look into that -- mine isn't set up for that presently so I'll see what I need to change